core/src/banking_stage.rs

```diff
@@ -577,7 +582,7 @@ impl BankingStage {
         let result = bank.check_transactions(
             &transactions,
             &filter,
-            MAX_RECENT_BLOCKHASHES / 2,
+            (MAX_RECENT_BLOCKHASHES - MAX_TRANSACTION_FORWARDING_DELAY) / 2,
```

Comment:
This will shorten the buffer for non-forwarded transactions as well right?

Reply:
Yes.